packages/cache/src/storage.integration.ts

```diff
@@ -32,6 +32,11 @@ export enum StoreError {
   NotEnoughSpace,
 }
 
+export interface StoreFileReturnValue {
```

Comment:
I have been thinking about this situation that happens quite often where we need to export interfaces. Because normally every interface and class should be in their own file. However, I realize that many times, interfaces like this one that are ONLY needed for the class or interface in the same file, does not make sense to move to a separate file, as that would in fact pollute the file structure and make it less clear for what a given interface is supposed to be used. Thoughts?

Reply:
I think that when an interface is used like this, as a type defining an in-parameter or return value of a function it should be defined in the same file as the function itself. Defining it in a separate file just makes the "real" interfaces hard to find among all the return types.